.env

```diff
@@ -7,3 +7,4 @@ SCIENCEBEAM_JUDGE_IMAGE=elifesciences/sciencebeam-judge_unstable:7eabb8b7e23d171
 SCIENCEBEAM_DEFAULT_LIMIT=10
 SCIENCEBEAM_WATCH_INTERVAL=60
 SCIENCEBEAM_GCP_PROJECT=elife-ml
+DOCKER_HUB_CREDENTIALS_URL=
```

Comment:
I could put the actual URL in here which would make it easier for me. It would reveal the name of the bucket, which would still be secured (not public) of course.

Reply:
if it makes it easier, tne the value is always overridable so `.env` is a good place for defaults

Reply:
Actually maybe not that much easier. I'll just put it in `.bashrc`. Feels slightly more secure not to make it public.